_includes/certs.html

```diff
@@ -1,6 +1,6 @@
 {% for cert in site.data.certs %}
 <div class="pe-lg-5 pt-lg-5 mt-lg-1 pb-5">
-    <h4 class="text-secondary pb-md-2">{{ cert.name }}</h4>
+    <span class="h4 text-secondary pb-md-2">{{ cert.name }}</span>
```

Comment:
Nice catch on the out-of-order headings.

I still think these should all be real heading elements though, looks like `<h2>` is probably the right level? Since this cert stuff is all important content we want to be high up in search results.

Reply:
It would be `<h2>` level for the home page, but since these certs are in the Footer, which is on every page - the `<h2>` would not be accurate for the job list page or the job index page. So I'm not sure what would be the right heading element, unless we made different footers for every template.

Reply:
I think it is still accurate? Like you said the footer is on every page, and every page has an `<h1>`, so we can always have one or more `<h2>` in the footer.

Reply:
Figured it out! Fixed here: https://github.com/compilerla/compiler.la/pull/116/commits/bbeaa067915e2238a4bced6d5f237781b808fcae